orbmain/src/test/java/com/sun/corba/ee/impl/encoding/FVDBuilder.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Distribution License v. 1.0, which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+package com.sun.corba.ee.impl.encoding;
+
+import com.sun.corba.ee.impl.corba.TypeCodeImpl;
+import com.sun.corba.ee.impl.util.RepositoryId;
+import com.sun.corba.ee.spi.orb.ORB;
+import com.sun.org.omg.CORBA.AttributeDescription;
+import com.sun.org.omg.CORBA.Initializer;
+import com.sun.org.omg.CORBA.OperationDescription;
+import com.sun.org.omg.CORBA.ValueDefPackage.FullValueDescription;
+import org.omg.CORBA.TCKind;
+import org.omg.CORBA.TypeCode;
+import org.omg.CORBA.VM_ABSTRACT;
+import org.omg.CORBA.VM_CUSTOM;
+import org.omg.CORBA.VM_NONE;
+import org.omg.CORBA.VM_TRUNCATABLE;
+import org.omg.CORBA.ValueMember;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
```

Comment:
A class description here of what the calls is doing would be good for the future

Reply:
Updated to add Javadoc.